### PR TITLE
Task3

### PR DESCRIPTION
**Pull request by GGnk**
## Evaluation criteria

1. Markup is done with React Components and React Fragments (parent-child); 
2. Apply styles (no need to do pixel perfect and strict colors following); 
3. Use PropTypes (Use Typescript); 
4. Use <ErrorBoundary> component for catching and displaying errors. 

### Demo - [web page](https://react-nurlan.netlify.app)

### CODE REVIEW

File: src/assets/app.scss [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+            width: fit-content;
+            line-height: 2;
+        }
+    }
+
+    .btn {
+        width: 176px;
+        height: 46px;
+        font-size: 22px;
+        background: #55555593;
```
Comment (mmarkelov) [inline src/assets/app.scss]: is it valid hex color?

File: src/assets/app.scss [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+        border-radius: 5px;
+
+        &:hover {
+            background: $color4;
+        }
+
+        &.search {
+            width: 230px;
+            height: 55px;
+            line-height: 2.5    ;
```
Comment (mmarkelov) [inline src/assets/app.scss]:
```js
line-height: 2.5;
```

File: src/components/Header.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import Search from './blocks/Search'
+
+const Header: React.FC = ({ children }) => {
+
+    return (
+        <header className='header'>
+            <div className="blur">
+                <div className="top">
+                    { children }
```
Comment (mmarkelov) [inline src/components/Header.tsx]: I'm not sure if you really need use `children`, you can just use `<Logo/>` here

Comment (GGnk) [reply src/components/Header.tsx]: Исправил

File: src/assets/app.scss (hunk 2 above)
Comment (GGnk) [reply src/assets/app.scss]: Исправил

File: src/assets/app.scss (hunk 1 above)
Comment (GGnk) [reply src/assets/app.scss]: Исправил

File: src/components/blocks/MovieCard.tsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Movie } from '../../assets/js/types'
+
+const Search: React.FC<Movie> = ({ id, title, year, genres, posterurl }) => {
+    
+    const listGenres = genres.map((item: string, i) => {
```
Comment (mmarkelov) [inline src/components/blocks/MovieCard.tsx]: Not an error, but you can use `join`:
```js
genres.join(' & ')
```

Comment (GGnk) [reply src/components/blocks/MovieCard.tsx]: I even forgot something, thank you)

**Pull request closed without merging**